### PR TITLE
HSEARCH-2322 Test case and fix

### PR DESCRIPTION
**Pull request by yrodiere**
See https://hibernate.atlassian.net/browse/HSEARCH-2322

### CODE REVIEW

File: elasticsearch/src/test/java/org/hibernate/search/elasticsearch/testutil/OrmTestRunner.java [hunk 1]
```diff
@@ -20,7 +20,7 @@
  * @author Gunnar Morling
  */
 @RunWith(ClasspathSuite.class)
-@ClasspathSuite.ClassnameFilters({ ".*ProjectionQueryTest" })
-// @ClasspathSuite.ClassnameFilters({ "org.hibernate.search.test.*" })
+//@ClasspathSuite.ClassnameFilters({ ".*ProjectionQueryTest" })
+ @ClasspathSuite.ClassnameFilters({ "org.hibernate.search.test.jpa.SpatialQueryingJPATest" })
 public class OrmTestRunner {
```
Comment (gsmet) [inline elasticsearch/src/test/java/org/hibernate/search/elasticsearch/testutil/OrmTestRunner.java]: Please revert these changes, they are useless.

Comment (yrodiere) [inline elasticsearch/src/test/java/org/hibernate/search/elasticsearch/testutil/OrmTestRunner.java]: Yes, this was a mistake. Fortunately, this broke the build.
I'll make sure to only add the "Ready for review" tag when the build passes, in the future... Sorry for the inconvenience.

File: orm/src/test/java/org/hibernate/search/test/jpa/SpatialQueryingJPATest.java [hunk 2]
```diff
@@ -226,6 +227,63 @@ public void testDistanceSort2() throws Exception {
...
+		FullTextQuery hibQuery = em.createFullTextQuery( luceneQuery, POI.class );
+		Sort distanceSort = new Sort( new DistanceSortField( centerLatitude, centerLongitude, "location" ) );
+		hibQuery.setSort( distanceSort );
+		hibQuery.setProjection( FullTextQuery.THIS, FullTextQuery.SPATIAL_DISTANCE );
+		hibQuery.setSpatialParameters( centerLatitude, centerLongitude, "location" );
+		List results = hibQuery.getResultList();
+		Object[] firstResult = (Object[]) results.get( 0 );
+		Object[] secondResult = (Object[]) results.get( 1 );
+		Assert.assertEquals( 10.1582, (Double) firstResult[1], 0.01 );
+		// ElasticSearch returns a distance of Double.MAX_VALUE while Lucene just returns the distance to (0,0)
```
Comment (gsmet) [inline orm/src/test/java/org/hibernate/search/test/jpa/SpatialQueryingJPATest.java]: Could you add a reference to HSEARCH-2324 in both comments so that we can keep track of it.

Other than that it looks good to me.

Thanks!

Comment (yrodiere) [inline orm/src/test/java/org/hibernate/search/test/jpa/SpatialQueryingJPATest.java]: I just did. I'll rebase and merge, then. Thanks!
